### DISCUSSION TITLE
Pagination in a view

### DISCUSSION THREAD

**Discussion by kevh-c** [twilio-labs/paste#905] [Q&A] [answered]
**Problem**
We're building an improvement on a feature where a customer can select multiple services in a list to auto-create a conversation. Our unknown is that customers typically only have a small handful of services (less than 10) but in theory they can make as many as they want. 

**Proposal**
To add pagination if messaging services in an account are more than 10. At the moment I'm using a card but understand this could be a pretty bespoke solution so it would be built with a Box primitive. 

Example screenshot:
![configure 04](https://user-images.githubusercontent.com/2405518/97993067-7735f400-1ddb-11eb-91c8-12f2f50c6d04.png)

For more context the current prototype can be found [here](https://twilio.invisionapp.com/share/JKZ8COX28DH#/screens/435899532)(select Design on the cover).

**Question**
I'd love to get your thoughts on this approach. Have the team seen anything similar on Console? Are there any recommendations which support (or don't support) this approach?

Thanks!

**Comment by corinne-nagel** [comment]
Hi @kevhealy-git I'll take this to the team and get back to you with some ideas. Currently our time to respond is a few days. Thanks!

**Reply by kevh-c** [reply]
Thanks Corinne appreciated!

**Comment by corinne-nagel** [comment, marked as answer]
Hi again @kevhealy-git , after talking with the team, this seems like something that could be in a scrollable box rather than paged (the paging feels a little off here/perhaps there should be a clearer view of what's selected). Have you tested this experience with users yet or are you planning to?

**Reply by kevh-c** [reply]
Thanks for the feedback Corinne — I agree that the hiding of selected items because of pagination makes things less clear. A scrollable box makes things a lot simpler. I will have a look at this approach. 
This hasn't been tested yet but we are intending to in the next few sprints. I will aim to test with a scrollable box and see how we get on.

**Reply by corinne-nagel** [reply]
Sounds great, look forward to hearing how it goes!
